utils/main.go

```diff
@@ -200,6 +203,9 @@ func (u *utils) SendContractTransaction(
 
 	opts.NoSend = false
 	opts.GasLimit = tx.Gas() * gasLimitBumpRatio / Percentage
+	if opts.GasLimit > maxGasLimit {
+		opts.GasLimit = maxGasLimit
```

Comment:
if we hit this case, the transaction  still run well with forcing gas to 50.000.000?

Reply:
We will adjust to bulk task logic to avoid high gas transaction too